Developers/Newcomers_Guide.rst

```diff
@@ -149,6 +149,11 @@ clicking ``New Pull Request``.
 
 **Congratulations!** You have now created your first ``Pull Request``!
 
+.. note::
+
+    Do not delete your comments on Github because it makes it hard for other
+    developers to follow on that issue. If necessary, edit your comment.
```

Comment:
I'd discourage editing as well unless it's like task lists or something really minor like a typo. Whenever adding new info, make a new comment so notifications are correct.